templates/hanadb_exporter.j2

```diff
@@ -1,30 +1,20 @@
-{%- from "hana/map.jinja" import hana with context -%}
-{% set host = grains['host'] %}
-
-{# sid is coming from exporter.sls context #}
-{% for node in hana.nodes if node.host == host and node.exporter is defined and node.sid == sid %}
-
-{% set dbinst = '{:0>2}'.format(node.instance) %}
-{% set node_subix = '{}_{}'.format(node.sid, dbinst) %}
-
+{# context is coming from monitoring.sls #}
 {
-  "exposition_port": {{ node.exporter.exposition_port|default(9668) }},
-  "multi_tenant": {{ node.exporter.multi_tenant|default(true)|tojson }},
-  "timeout": {{node.exporter.timeout|default(600)}},
+  "exposition_port": {{ exporter.exposition_port|default(9668) }},
+  "multi_tenant": {{ exporter.multi_tenant|default(true)|tojson }},
+  "timeout": {{exporter.timeout|default(600)}},
   "hana": {
-    "host": "{{ node.host }}",
-    {% if node.exporter.multi_tenant|default(true) is sameas true and node.exporter.port is not defined %}
-    "port": 3{{ dbinst }}13,
-    {% else %}
-    "port": {{ node.exporter.port }},
-    {% endif %}
-    "user": "{{ node.exporter.user }}",
-    "password": "{{ node.exporter.password }}"
+    "host": "{{ grains['host'] }}",
```

Comment:
This should be just `{{ host }}`, right?
This is why we are setting the value in the context `hanadb_exporter_configuration_{{ exporter_instance }}`

Reply:
actually `host` will always be equal to `grains['host']`; I forgot to remove the additional variable from the context.

Reply:
Ok, remove this or to other, but both shouldn't be there hehe